version/version.go

```diff
@@ -53,7 +56,9 @@ func Version() Info {
 		Version:   version,
 		Branch:    branch,
 		Revision:  revision,
-		GoVersion: goVersion,
```

Comment:
Should probable be a TODO to delete this global or mark it as deprecated? I understand not wanting to break any scripts still setting it.

Reply:
I swapped it runtime.Version(), which seems a little superfluous with calling runtime. but fits nicely with the `PrintFull` function. :shrug: Not sure this really gets any use

Reply:
My point is that the `goVersion` variable in the var block is now unused.